platform-sdk/consensus-otter-docker-app/src/testFixtures/java/org/hiero/consensus/otter/docker/app/DockerInit.java

```diff
@@ -39,6 +47,23 @@ public final class DockerInit {
     /** Lazily initialised docker app; only set once. */
     private DockerApp app;
 
+    private final AtomicBoolean cancelled = new AtomicBoolean(false);
```

Comment:
I think using a volatile boolean is sufficient.

Reply:
check